docs/src/styles/docs/sidebar.scss

```diff
@@ -134,6 +134,13 @@
   }
 }
 
+// FF 
+// @-moz-document url-prefix() {
```

Comment:
Is this comment still needed?

Reply:
I think Danny was the one who originally commented it out. Not sure if I should've removed it or not since he was still trying some things out.

If you want I can remove it though!